### PR TITLE
Modifications for ATM nems.configure prep.

### PR DESCRIPTION
**Pull request by HenryRWinterbottom**
**Description**

This PR contains method to build a `AttrDict` attribute containing the configuration variables relevant to the `ATM` application of the global-workflow. 

Subsequent PRs will update the configuration as necessary.

**Type of change**

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

**How Has This Been Tested?**

The expected attributes have been defined with the corresponding `AttrDict` corresponding to the global-workflow `ATM` configuration.
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: ush/python/pygfs/ufswm/ufs.py [hunk 1]
```diff
@@ -205,3 +206,16 @@ def mdl_config(self) -> None:
...
+        """
+        Description
+        -----------
+
+        This method prepares and builds the `nems.configure` file for
+        the UFS forecast.
+
+        """
+
+        cfg =
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/ufs.py]: probably want to `pass` here.

File: ush/python/pygfs/task/gfs_forecast.py [hunk 2]
```diff
@@ -53,8 +53,10 @@ def initialize(self) -> None:
         self.gfs.stage_fix()
 
         # Stage the different tables
-        self.gfs.stage_tables(table='diag_table', target=os.path.join(self.task_config.DATA, 'diag_table.tmpl'))
-        self.gfs.stage_tables(table='field_table', target=os.path.join(self.task_config.DATA, 'field_table'))
+        self.gfs.stage_tables(table='diag_table', target=os.path.join(
+            self.task_config.DATA, 'diag_table.tmpl'))
+        self.gfs.stage_tables(table='field_table', target=os.path.join(
+            self.task_config.DATA, 'field_table'))
```
Comment (aerorahul) [inline ush/python/pygfs/task/gfs_forecast.py]: I am curious about these and some more below.
Is something in your IDE that is forcing this change?
To me this makes the code less readable.

File: ush/python/pygfs/ufswm/gfs.py [hunk 3]
```diff
@@ -52,16 +52,22 @@ def build_ufs_model(self) -> None:
...
+        self.ufs_model.fix.FIX_aer = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'aer')
+        self.ufs_model.fix.FIX_am = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'am')
+        self.ufs_model.fix.FIX_lut = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'lut')
+        self.ufs_model.fix.FIX_orog = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'orog')
+        self.ufs_model.fix.FIX_ugwd = os.path.join(
+            self.ufs_model.HOMEufs, 'fix', 'ugwd')
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/gfs.py]: same comment as above.
The line-length set in this project is 120 characters.  And I am certain this is well below that mark.
So, why this change?

File: ush/python/pygfs/ufswm/gfs.py [hunk 4]
```diff
@@ -52,16 +52,22 @@ def build_ufs_model(self) -> None:
         # Get model dates relevant for this run
         self._get_date_info()
 
-        logger.debug(f"UFS-weather-model configuration:\n{pformat(self.ufs_model)}")
+        logger.debug(
+            f"UFS-weather-model configuration:\n{pformat(self.ufs_model)}")
```
Comment (aerorahul) [inline ush/python/pygfs/ufswm/gfs.py]: here too.

**Review by aerorahul — CHANGES_REQUESTED** [review]
Other than the error in the method to `nems_build`, some comments regarding reformating.

File: ush/python/pygfs/task/gfs_forecast.py [hunk 5]
```diff
@@ -7,7 +7,7 @@
 from pygw.task import Task
 from pygfs.ufswm.gfs import GFS
 
-logger = logging.getLogger(__name__.split('.')[-1])
+logger = logging.getLogger(__name__.split(".")[-1])
```
Comment (aerorahul) [inline ush/python/pygfs/task/gfs_forecast.py:10]: I will remember to use `"` next time onwards so that this does not show up as a change from others.
Though, then we may need to make it a rule for everyone.

Comment (HenryRWinterbottom) [reply ush/python/pygfs/task/gfs_forecast.py:10]: @aerorahul This is another artifact from `black`. No need to enforce, we can just push it through the `black` linter  pre-commit. Also, not need to bother others. I can make these changes incrementally in PRs.

**Review by aerorahul — APPROVED** [review]
looks good.
